### PR TITLE
Update with changes from latest AWS bundle update

### PR DESCRIPTION
**Pull request by larsll**

### CODE REVIEW

**Comment by larsll** [comment]
This has not yet been tested. Should be tested before merging.

**Comment by larsll** [comment]
@mattcamp -- do we want to progress on this? Do we want to remove something?

File: bundle/deepracer_simulation_environment/lib/deepracer_simulation_environment/download_params_and_roslaunch_agent.py [hunk 1]
```diff
@@ -45,6 +48,28 @@ def main():
         s3_yaml_name = sys.argv[4]
         launch_name = sys.argv[5]
 
+        LOG.info('S3 region {}'.format(s3_region))
+         # create boto3 session/client and download yaml/json file
+         session = boto3.session.Session()
++        ec2_client = session.client('ec2', s3_region)
++        LOG.info('Checking internet connection...')
++        response = ec2_client.describe_vpcs()
```
Comment (mattcamp) [inline bundle/deepracer_simulation_environment/lib/deepracer_simulation_environment/download_params_and_roslaunch_agent.py]: I think we will need to remove or bypass this new code for running in non-AWS environments.

**Comment by mattcamp** [comment]
> @mattcamp -- do we want to progress on this? Do we want to remove something?

Definitely need to bypass some of the new code in download_params_and_roslaunch_agent.py (the part that checks the VPC for internet access) however I think that's the only thing that would cause us a problem (from my brief review of the changes)

File: bundle/deepracer_simulation_environment/lib/deepracer_simulation_environment/download_params_and_roslaunch_agent.py (hunk 1 above)
Comment (larsll) [reply bundle/deepracer_simulation_environment/lib/deepracer_simulation_environment/download_params_and_roslaunch_agent.py]: I have now changed it (still needs to be tested) that has one way of working in Local (S3_ENDPOINT_URL defined) and one in AWS.

Comment (mattcamp) [reply bundle/deepracer_simulation_environment/lib/deepracer_simulation_environment/download_params_and_roslaunch_agent.py]: Looks good to me. Merge at your leisure :)

**Pull request merged**